### PR TITLE
Router add ons

### PR DESCRIPTION
**Pull request by StefanRudareanu**

### CODE REVIEW

**Review by CatalinTiu — APPROVED** [review]
looks good, need to solve the conflicts as well and all's good in my opnion

File: src/components/TabletApp/RightSide/MeetingInfo/MeetingInfo.tsx [hunk 1]
```diff
@@ -26,47 +27,85 @@ const MeetingInfo = () => {
...
+                            providing an intuitive on-stage interface for both
+                            technical and non-technical personnel. With features
+                            such as redundant power supply for maximum
+                            reliability, easy transportability, and dozens of
+                            mounting options, the Terno is perfect for any live
+                            event. Product Features: - Modular design makes
+                            setup quick and simple. - 60 W amplifier delivers
+                            quality sound with clear, defined mid frequencies. -
+                            8 channels (2 each of omni/ambient/speaker 1/2)
+                            provide ample sound coverage for larger venues. -
```
Comment (Liubzzz) [inline src/components/TabletApp/RightSide/MeetingInfo/MeetingInfo.tsx:100]: here

File: src/components/TabletApp/RightSide/MeetingInfo/MeetingInfo.tsx [hunk 2]
```diff
@@ -6,14 +6,15 @@ const MeetingInfo = () => {
         "Stefan Rudareanu",
         "Andrei Cineva",
         "Cosmin Liuba",
-        "Stefan Rudareanu",
+        "Andrei Toba Daniel",
         "Andrei Cineva",
         "Cosmin Liuba",
```
Comment (Liubzzz) [inline src/components/TabletApp/RightSide/MeetingInfo/MeetingInfo.tsx:11]: here as well

**Review by Liubzzz — APPROVED** [review]

**Review by cristiberzescu — APPROVED** [review]
looks good

**Pull request merged**